src/model/game.py

```diff
@@ -45,23 +45,10 @@ class Game(QObject):
 
     SENTINEL = object()
 
-    def __init__(self,
-                 playerset,
-                 uid,
-                 state,
-                 launched_at,
-                 num_players,
-                 max_players,
-                 title,
-                 host,
-                 mapname,
-                 map_file_path,
-                 teams,
-                 featured_mod,
-                 featured_mod_versions,
-                 sim_mods,
-                 password_protected,
-                 visibility):
+    def __init__(self, playerset, uid, state, launched_at, num_players,
```

Comment:
Extra lines don't hurt us, and I think previous presentation was more readable.

Reply:
speak for yourself, they hurt me. I like to see more.
But if you insist, I'll undo it.